amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/Scheduler.java

```diff
@@ -953,6 +971,17 @@ void cleanupShardConsumers(Set<ShardInfo> assignedShards) {
         }
     }
 
+    private void cleanupLeaseCleanupManagers(Set<StreamIdentifier> assignedStreams) {
+        for (StreamIdentifier streamIdentifier : streamLeaseCleanupManagerMap.keySet()) {
+            if (!assignedStreams.contains(streamIdentifier)) {
+                streamLeaseCleanupManagerMap.remove(streamIdentifier);
+                log.debug("Removed lease cleanup manager for {} as this worker is no longer processing shards for that stream.",
```

Comment:
Always log the streamIdentifier, as streamNames can be same across accounts

Reply:
Removed these logs, but yeah good callout.